### PR TITLE
fix stickers

### PR DESCRIPTION
**Pull request by UhiOfficial**
file_unique_id instead of file_id

### CODE REVIEW

File: axol/config.py [hunk 1]
```diff
@@ -51,67 +52,61 @@
...
        'https://youtu.be/0ZF3rm86Ws4', 'https://youtu.be/3AZ5f_4_t1o', 'https://youtu.be/kZ2V8tJ-Q4c',
        'https://youtu.be/j6rrI6X0to4', 'https://youtu.be/JaciHAcvlyA', 'https://youtu.be/1Vxet85Bk0c',
        'https://youtu.be/L4H4uy9DAK4', 'https://youtu.be/QtnM-igAaTE', 'https://youtu.be/XvVij3q0sk0',
-       'https://youtu.be/7OaoxICcBoM', 'https://youtu.be/k4UU-iHLc1s', 'https://youtu.be/5WxHF_NTnC8'], []],
+       'https://youtu.be/7OaoxICcBoM', 'https://youtu.be/k4UU-iHLc1s', 'https://youtu.be/5WxHF_NTnC8'],
+        ['CAACAgQAAx0CQgMQgQABAYl4XvUIi9fLohxAaOtNNNMAAWDCmYO-AAKZAAMVdukH9Btqq0RjzZgaBA']],
     [['D: D: D: D: D: D: D: D: D: D:'], [], 0, [], ['CAADAgADHQADjW7LE5T5heR8tu6uAg']],
-    [[], ['CAADAgADHQADO9HBD8DTsJ6PcoXXAg'], rinneko_chat, ['О, РИНЕЙКА.\nЗАКИДЫВАТЬ МОЛНИЯМИ!',
+    [[], ['AgADHQADO9HBDw'], rinneko_chat, ['О, РИНЕЙКА.\nЗАКИДЫВАТЬ МОЛНИЯМИ!',
                                                             'О, РИННЕКО.\nЗАКИДЫВАТЬ МОЛНИЯМИ!',
```
Comment (Deepwarrior) [inline axol/config.py:80]: Could you additionally fix indentation?

File: axol/config.py [hunk 2]
```diff
@@ -51,67 +52,61 @@
...
 
     [['АНТИКЛАЦ!'], [], 0, [], [], "anti_task"],
-    #[[], ['CAADAgADnAADg8cnESB0jVNnakYoAg'], 0, [], [], "whois"],
+    #[[], ['AgADnAADg8cnEQ'], 0, [], [], "whois"],
     [["ПЕРЕНОСИМ"], [], 0, [], [], "stop_natalka"],
-    [[], ['CAADAgADLgADsjRGHsGbRuS2PA26Ag'], 0, [], [], "kick_citrus"],
-    [[], ['CAADAgADKAADO9HBD1SvkhGZJT_BAg', 'CAADAgADVwADO9HBD35E69ryx07IAg'], 0, [], [], "kick_rels"],
-    [[], ['CAADAgADLwADsjRGHvSXP4ESf3tIAg'], 0, [], [], "kick_misha"],
+    [[], ['AgADLgADsjRGHg'], 0, [], [], "kick_citrus"],
+    [[], ['AgADKAADO9HBDw', 'CAADAgADVwADO9HBD35E69ryx07IAg'], 0, [], [], "kick_rels"],
```
Comment (Deepwarrior) [inline axol/config.py:103]: Please, fix second sticker or just remove it

File: axol/config.py (hunk 1 above)
Comment (UhiOfficial) [reply axol/config.py]: Done

File: axol/config.py (hunk 2 above)
Comment (UhiOfficial) [reply axol/config.py]: Done

**Comment by Deepwarrior** [comment]
The best PR during last months!

**Pull request merged**